data/mods/Tanks/items.json

```diff
@@ -0,0 +1,115 @@
+[
+    {
+        "type":"GENERIC",
+        "id" : "motor_electric_super",
+        "name" : "supercharged electric motor",
+        "description" : "A massive, high-efficiency electronic motor designed for atomic-engine military vehicles.",
+        "weight" : 73708,
+        "to_hit" : 0,
+        "color" : "light_cyan",
+        "symbol" : ",",
+        "material" : ["superalloy", "null"],
+        "volume" : 15,
+        "bashing" : 9,
+        "cutting" : 0,
+        "category" : "veh_parts",
+        "price" : 680000
+    },
+    {
+        "type" : "AMMO",
+        "id" : "120mm_HEAT",
+        "price" : 100000,
+        "name" : "120mm HEAT",
+        "symbol" : "=",
+        "color" : "dark_gray",
+        "description" : "A 120mm High Explosive Anti Tank round.  It could ruin anyone's whole day.",
+        "material" : ["steel", "powder"],
+        "volume" : 10,
+        "weight" : 2000,
+        "bashing" : 1,
+        "cutting" : 0,
+        "to_hit" : 0,
+        "ammo_type" : "120mm",
+        "casing" : "NULL",
+        "damage" : 4000,
+        "pierce" : 800,
+        "range" : 80,
+        "dispersion" : 5,
+        "recoil" : 30,
+        "count" : 1,
+        "effects" : ["COOKOFF", "EXPLOSIVE", "TRAIL"]
+    },
+  {
+    "id": "tank_gun_manual",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "green",
+    "name": "120mm tank gun",
+    "name_plural": "120mm tank gun",
+    "description": "A 120mm cannon from a tank.  Obviously it needs to be mounted on a vehicle to fire.",
+    "price": 90000000,
+    "material": "hardsteel",
+    "flags": ["NEVER_JAMS", "MOUNTED_GUN"],
```

Comment:
As far as I recall, "MOUNTED_GUN" does nothing to prevent using it without having a vehicle attached.